### DISCUSSION TITLE
Verifying Result ERROR REEBOOTING

### DISCUSSION THREAD

**Discussion by nickforev** [RavenSystem/esp-homekit-devices#2227] [General]
Hi. I have old switches on ESP8266, with firmware HAA 11.9.1 I replaced the router and they did not want to update.
Removed, reflashed to current HAA 12.12.3, but in the end the problem is Verifying Result ERROR.
I thought it was a problem with the Internet, but switching to mobile Internet did not change anything.
In addition, I found very old sonoff basic R2 relays with HAA 2.3.7 firmware, wiped and physically flashed to 12.12.3 and in the end the problem is Verifying, Result ERROR, REEBOOTING.
When flashing new devices and devices on ESP8285, there is no such problem. Log added.

Also, I cannot find information anywhere, it is possible to flash the offline version of the HAA firmware and what range of memory addresses is required for different files. Earlier on ravencore, flash files with option: esptool.py write_flash -fs 4MB -fm dout -ff 40m 0x0 rboot.bin 0x1000 blank_config.bin 0x2000 otaboot.bin

[Mar 05 151400 HAA Log.log](https://github.com/RavenSystem/esp-homekit-devices/files/14496354/Mar.05.151400.HAA.Log.log)

**Comment by nickforev** [comment]
Add...
Same Wi-Fi, other device, result ok. ![image](https://github.com/RavenSystem/esp-homekit-devices/assets/67782540/62222518-3abc-4b8d-a474-0c9c238e0dfa)![image](https://github.com/RavenSystem/esp-homekit-devices/assets/67782540/35967f66-5128-413c-8eb0-0cf55bcc691b)
!
[Mar 05 163347 HAA Log.log](https://github.com/RavenSystem/esp-homekit-devices/files/14496806/Mar.05.163347.HAA.Log.log)

**Reply by RavenSystem** [reply]
Damaged flash chip.

**Comment by nickforev** [comment]
Maybe damaged, but i have backup file, and after recovery original file, ewelink work ok.
I have others devices with same problem, before HAA 12 update (changes amazon server), they all works good.

It's only replace other clear chip or maybe program fix, full erase, flash blank.bin?

**Reply by xrust83** [reply]
Replace the memory chip. Probably a damaged sector where ESP cannot write or read. I have such a Sonoff Basic, one of my first boards, which I flashed up to 100 times only through the cable. Tasmota is installed and running on the board without any problems, but HAA cyclically overloads when accessing Wi-Fi, for example. Replacing the ESP did not give results. Replaced the memory chip with a WINBOND 25Q16 (2 Mb) HAA was installed on the first try. Before that, Jose bothered me, he made changes to the firmware, increased the timings of the firmware, etc.

![image](https://github.com/RavenSystem/esp-homekit-devices/assets/58448955/70e2b664-41ea-4f55-aed5-0e69ed123af2)
